mmv1/templates/terraform/pre_delete/oracledatabase_cloud_exadata_infrastructure.go.tmpl

```diff
@@ -0,0 +1,3 @@
+if d.Get("deletion_protection").(bool) {
+	return fmt.Errorf("cannot destroy instance without setting deletion_protection=false and running `terraform apply`")
```

Comment:
```suggestion
	return fmt.Errorf("cannot destroy google_oracle_database_cloud_exadata_infrastructure resource %q without setting deletion_protection=false and running `terraform apply`", d.Id())
```

Let's make the error message more specific to the resource, and call out which it is.

(may not run as written, but should be easy to fix if it doesn't)